genomicFileHandler/pileup_reader.py

```diff
@@ -1,57 +1,53 @@
 #!/usr/bin/env python3
 
-import regex as re
-import math
-import sys, os, gzip
-import genomic_file_handlers as genome
+import sys, os, gzip, re, math
 
 nan = float('nan')
 inf = float('inf')
 
-
 def seq(reads):
-    
+
     '''A function to convert a string into a character generator.'''
     for read_i in reads:
         yield read_i
 
 
 class Pileup_line:
-    
+
```

Comment:
Is this reading the output of `samtools pileup`?

Reply:
Not a part of somaticseq anymore. It's only called in a legacy script `utilities/attach_pileupVAF.py`, which attaches VAF values calculated from pileup files to any VCF input file.

Reply:
Thanks. Can you add a comment before to indicate that this part of the code is deprecated?